src/goose/brokers/redis/api/enqueued_jobs.clj

```diff
@@ -18,15 +18,40 @@
         match? (fn [job] (= (:id job) id))]
     (first (find-by-pattern redis-conn queue match? limit))))
 
-(defn prioritise-execution [redis-conn job]
-  (let [ready-queue (:ready-queue job)]
-    (when (redis-cmds/list-position redis-conn ready-queue job)
-      (redis-cmds/del-from-list-and-enqueue-front redis-conn ready-queue job))))
+(defn prioritise-execution
+  ([redis-conn job]
+   (let [ready-queue (:ready-queue job)]
+     (when (redis-cmds/list-position redis-conn ready-queue job)
+       (redis-cmds/del-from-list-and-enqueue-front redis-conn ready-queue job))))
 
-(defn delete [redis-conn job]
-  (let [queue (job/ready-or-retry-queue job)]
-    (= 1 (redis-cmds/del-from-list redis-conn queue job))))
+  ;; This isn't exposed by Broker API and is used internally by console
+  ([redis-conn queue jobs]
```

Comment:
Note: I have used multi-arity functions instead of defining in a separate function like prioritise-execution-multiple.